rfcs/text/0352-graph-type-system.md

```diff
@@ -783,7 +783,7 @@ This RFC defines the following primitive top-level Data Types
   ```json
   {
     "kind": "dataType",
-    "$id": "https://blockprotocol.org/types/@blockprotocol/data-type/text",
+    "$id": "https://blockprotocol.org/@blockprotocol/types/data-type/text",
```

Comment:
I'm assuming version numbers are intentionally omitted

Reply:
Yes, versioning came in an RFC after, I'm just patching the previous one to match the current URL format because it'll be less confusing in posterity